src/main/java/org/jabref/logic/openoffice/oocsltext/CSLCitationOOAdapter.java

```diff
@@ -12,55 +14,167 @@
 import org.jabref.model.openoffice.ootext.OOFormat;
 import org.jabref.model.openoffice.ootext.OOText;
 import org.jabref.model.openoffice.ootext.OOTextIntoOO;
-import org.jabref.model.openoffice.uno.CreationException;
 
-import com.sun.star.lang.WrappedTargetException;
 import com.sun.star.text.XTextCursor;
 import com.sun.star.text.XTextDocument;
 import org.apache.commons.text.StringEscapeUtils;
 
 public class CSLCitationOOAdapter {
 
+    public static final String[] PREFIXES = {"JABREF_", "CSL_"};
```

Comment:
Is the prefix `CSL_` really needed? I think, only `JABREF_` is written as prefix.